### ISSUE TITLE
Support LiveJournal.com?

### ISSUE THREAD

**Issue by abartov** [eloquence/freeyourstuff.cc#64]
I just posted a review that was too long for goodreads.com, so I resorted to my ancient LJ account, but it would be nice to be able to share to freeyourstuff.cc.

OTOH LJ is mostly dead (outside the Russosphere, at least), so probably not a high priority.

**Comment by brainwane** [comment]
Given [recent events](https://www.metafilter.com/166250/LiveJournal-now-bans-political-talk-LGBT-talk) people's interest in getting at their LiveJournals without having to agree to the new Terms of Service (via [user styles](https://gist.github.com/ivymyt/c1496b83bfce7733e43a1a0db07e6916), [Selenium](https://xtina.dreamwidth.org/4294792.html), & other tools) has risen. I see a bunch of people, for instance in that MetaFilter thread, unhappy about current LJ archiving options. A freeyourstuff plugin would be great.

**Comment by abartov** [comment]
Indeed.  I recently wanted to back up my ancient LJ, and none of the many tools written over the years still worked.  After much searching, I landed on https://gregstoll.com/ljbackup/ so I'm recording it here for anyone else's benefit.

**Comment by eloquence** [comment]
Thank you for the report & links. Does it still make sense to support this in FYS given existing export options? What would an FYS plugin have to add to be of value? (I'll note that it wouldn't have to ask the user for their password, at least.)

The evil TOS acceptance dialog looks like it would be easy enough to ignore in its current implementation. It's difficult for me to test a full flow with TOS dialog -- my account is generating only internal server errors (it may be too old, and I don't think it has any posts to it). I'd need an account with posts which hasn't accepted the TOS to work on a full export flow.
